pkgs/xed/default.nix

```diff
@@ -10,24 +10,17 @@ stdenv.mkDerivation rec {
     sha256 = "1757anm01nl0b7nqyh2bvgdrz5fhgvdyjpcx2nvg05v8icdj2xmh";
   };
 
-  postUnpack = ''
-    t="$(ls -d xed-*)/mbuild"
-    cp -ar ${mbuild} $t
-    chmod u+w -R $t
-  '';
-
-  nativeBuildInputs = [ python ];
-
-  buildPhase = ''
-    python ./mfile.py -j$NIX_BUILD_CORES
-  '';
+  nativeBuildInputs = [ mbuild ];
 
   installPhase = ''
-    python ./mfile.py -j$NIX_BUILD_CORES --prefix=$out install
+    python ./mfile.py install --shared --install-dir=$out
+    ln -s $out/include/xed/* $out/include
```

Comment:
Intel guys are using it slightly different: https://github.com/01org/processor-trace/blob/master/ptxed/src/ptxed.c#L47

Reply:
Hmm, interesting.

Did some digging and found this: https://github.com/intelxed/xed/commit/fcd2d41b2bd8802eed4e620a6836795fbd7ad919#diff-4c830633847d5e1220d61fab27a0e28a

So I think it should be left as-is, and do as they say re:adjusting include path or fixing code.